lib/wallaby/dsl/finders.ex

```diff
@@ -13,6 +15,17 @@ defmodule Wallaby.DSL.Finders do
     do_find_elements(session, to_params(query))
   end
 
+  def wait_until_visible(session, query, opts \\ [])
+  def wait_until_visible(session, query, [{:count, count} | _] = _opts) do
+    retry fn ->
+      case all(session, query) do
+        elements when length(elements) == count -> elements
```

Comment:
I was thinking count: :any would wait until one or more elements were visible, or timeout. The use case I ran into was checking that an employee was on the bench. Depending on the size of the window, a different number of weeks are visible. So if I do `wait_until_visible(".bench__availability", count: :any)` I get some variable number of nodes. Alternatively, a more specific selector would solve my problem, but I'm not sure what that'd look like. I can't do `.bench__availability:first`. Maybe an xpath? Haven't used those before